app/views/hiring_staff/vacancies/job_summary/show.html.haml

```diff
@@ -22,9 +22,10 @@
         rows: 10,
         required: true
 
+      - organisation = @vacancy.school_or_school_group.is_a?(SchoolGroup) ? 'Trust' : 'School'
```

Comment:
...Third.

https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Given similar logic in three places, let's put this logic into the `OrganisationHelper` and reuse it in each of these three places.

(In the other two instances, there was e.g. `'trust_visits'` rather than `'Trust'`, but you can interpolate `_visits` and `.downcase` the strings quite simply in the views.)

Reply:
Done!!